### ISSUE TITLE
Easter egg addons

### ISSUE THREAD

**Issue by RedGuy12** [ScratchAddons/addons-data-updater#2]
Is this supposed to include the easter egg addons? Because it looks like it does.

**Comment by WorldLanguages** [comment]
Yeah, maybe those shouldn't be considered

**Comment by Hans5958** [comment]
Mind telling what's wrong with including them on the data? I could hide it on the website, but you seem to create the issue here?

**Comment by WorldLanguages** [comment]
@Hans5958 Yeah you could hide them in the website as well (and maybe show them after pressing konami code, hehe)
